### DISCUSSION TITLE
训练报GPUpaddle不兼容怎么办？跪求大佬帮忙看一下

### DISCUSSION THREAD

**Discussion by 694578908** [PaddlePaddle/PaddleOCR#13372] [Q&A]
W0712 22:17:19.883967 15416 gpu_resources.cc:96] The GPU architecture in your current machine is Pascal, which is not compatible with Paddle installation with arch: 70 75 80 86 , it is recommended to install the corresponding wheel package according to the installation information on the official Paddle website.
W0712 22:17:19.883967 15416 gpu_resources.cc:119] Please NOTE: device: 0, GPU Compute Capability: 6.1, Driver API Version: 12.4, Runtime API Version: 12.0
W0712 22:17:19.887956 15416 gpu_resources.cc:149] device: 0, cuDNN Version: 8.9.
[2024/07/12 22:17:19] ppocr INFO: train dataloader has 5 iters
[2024/07/12 22:17:19] ppocr INFO: valid dataloader has 2 iters
[2024/07/12 22:17:19] ppocr INFO: train from scratch
[2024/07/12 22:17:19] ppocr INFO: During the training process, after the 0th iteration, an evaluation is run every 5 iterations
W0712 22:17:32.124197  8972 system_allocator.cc:231] cudaHostAlloc failed.
W0712 22:17:32.124197  8972 naive_best_fit_allocator.cc:387] cudaHostAlloc Cannot allocate 58982400 bytes in CUDAPinnedPlace

**Comment by UserWangZz** [comment]
GPU型号是什么？

**Reply by 694578908** [reply]
感谢，已解决

**Comment by GreatV** [comment]
换一个显卡

**Reply by 694578908** [reply]
感谢，已解决
